src/compiler/checker.ts

```diff
@@ -5782,6 +5740,20 @@ namespace ts {
             }
         }
 
+        // Return true if the given type is the constructor type for an abstract class
+        function isAbstractConstructorType(type: Type) {
+            if (type.flags & TypeFlags.Anonymous) {

```

Comment:
Can you add that in as a quick comment then? It's not entirely obvious at first glance.